CONTRIBUTING.md

```diff
@@ -130,25 +117,9 @@ If your pull request does two separate things, it should likely (but not
 necessarily) be multiple pull requests. We prefer for pull requests to be
 focused on one feature, issue, etc.
 
-We will not merge any pull requests that do not pass our CI suite, particularly
```

Comment:
I would replace this paragraph with something that spells out the general criteria we look for before merging PRs, which basically just consists of content and style. So maybe something like this?
> One of our teammates will review your pull request by locally evaluating the updated content, structure, and styling in your changes and making suggestions. They will manually check that your changes adhere to the general style of the repository, which at present does not have any automatic linting.